takwimu/models/dashboard.py

```diff
@@ -406,4 +408,12 @@ class Meta:
         verbose_name = 'About Us'
 
 
+@register_setting
+class LegalNoticeSetting(BaseSetting):
+    terms_of_use = RichTextField(blank=True, null=True)
+    privacy_policy = RichTextField(blank=True, null=True)
+    cookie_policy = RichTextField(blank=True, null=True)
+
+    class Meta:
+        verbose_name = 'Legal Notice'
```

Comment:
These have to be in code as earlier added by @nerdlet and not set in the settings because they are legal documents. Editing them must be hard.